src/test/groovy/org/shipkit/auto/version/AutoVersionTest.groovy

```diff
@@ -76,4 +78,22 @@ some commit #2
                 "  - reason: shipkit-auto-version caught an exception, falling back to reasonable default\n" +
                 "  - run with --debug for more info")
     }
+
+    def "uses project version if not default"() {
+        versionFile << "version=2.0.*"
+        runner.run("git", "tag") >> "v2.0.0"
+        runner.run("git", "log", "--pretty=oneline", "v2.0.0..HEAD") >> """
+some commit #1
+some commit #2
+"""
```

Comment:
I suspect we don't need this.

Reply:
While true it wouldn't change the test, I felt it was good to have it to verify it changed the behavior. I can get rid of it though.

Reply:
Done